server.js

```diff
@@ -32,7 +33,18 @@ app.get("/my-cv", (req, res) => {
   res.render("my-cv", { title: "Repos List From Github" });
 });
 app.get("/contact", (req, res) => {
-  res.render("contact", {title: "Welcome to Contact"});
+  res.render("contact", {title: "xxxxxWelcome to Contact"});
+});
+app.get("/posts", (req, res) => {
+  res.sendFile(filePath);
+});
+app.get("/posts/:id", (req, res) => {
+  if (postsJson ){
```

Comment:
It's better to load the posts from file here again. What if this endpoint gets hit but the loading code has not yet been run? postsJson will be null and this endpoint would not do it's job :)

Reply:
Ah! you have figured out the issue! I was thinking of asking you. I thought it was expected to have separate layout as there was a post.html when loading the post page. Your idea is good and simple. I will do it and let you know the outcome.